### ISSUE TITLE
Blueprint API 'populate where' returns empty array

### ISSUE THREAD

**Issue by Clan-Utility** [balderdashy/sails#3049]
As written in [doc](http://sailsjs.org/documentation/reference/blueprint-api/populate-where), we could get associated records using blueprint API /user/1/pets, but it returns an empty array.

Meanwhile, I could get those records using /user/1?populate=pets OR /pet?where={user:1}

Using sails.js v0.11.1

**Comment by Clan-Utility** [comment]
I've figured out that it was because, in blueprints/action/popluate.js, access_token was passed in where clause 

      var populate = sails.util.objCompact({
          where: where,
          skip: actionUtil.parseSkip(req),
          limit: actionUtil.parseLimit(req),
          sort: actionUtil.parseSort(req)
     });

**Comment by sailsbot** [comment]
Thanks for posting, @Clan-Utility.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
